reading-books-list-service-nodejs/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+displayName: Book Management Service in Java
+description: CRUD operations for books
+componentType: service
+buildPreset: nodejs
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /reading-books-list-service-nodejs
+# thumbnailPath: /reading-books-list-service-nodejs/icon.png
```

Comment:
Shall we add an icon for missing samples? We can use this icon for the time being - https://github.com/wso2/choreo-samples/blob/main/jwt-handling/icon.png

Please check on other places too.

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/15/commits/a67aed655be39d5f54333c8866d56f03c6d43989